### DISCUSSION TITLE
Python 3.12 type and generics syntax highlight issue

### DISCUSSION THREAD

**Discussion by kotazzz** [microsoft/pylance-release#6674] [Q&A]
![image](https://github.com/user-attachments/assets/7850409a-04d5-4090-9343-28b17d78f545)
I'm not sure where I should turn, any help would be appreciated. When and how will the backlighting be available? Maybe something I haven't updated or something I haven't turned on?

**Comment by rchiodo** [comment]
I have a feeling this is because the TextMate grammar shipped with VS code doesn't recognize the `type` keyword. You can tell it's confused here:

![image](https://github.com/user-attachments/assets/0df4fe2f-dd45-47dd-9847-1fea64d78d62)

See this issue which has been on our backlog for quite a while. 
https://github.com/microsoft/pylance-release/issues/5824

It's mostly waiting for this:
https://github.com/microsoft/vscode/issues/210475
